go/plugins/dotprompt/genkit.go

```diff
@@ -137,8 +140,16 @@ func (p *Prompt) Register() error {
 	if p.Variant != "" {
 		name += "." + p.Variant
 	}
-
-	p.action = ai.DefinePrompt("dotprompt", name, nil, p.buildRequest, p.Config.InputSchema)
+	// TODO: Unwind this change once Monaco Editor supports newer than JSON schema draft-07.
+	p.InputSchema.Version = ""
```

Comment:
I suggest a blank line before the TODO comment and after this line to clarify what "unwind this change" refers to.

Reply:
Done.